packages/utilities/psammead-test-helpers/README.md

```diff
@@ -10,6 +10,7 @@ This package provides a collection of helper methods for implementing Jest snaps
 | shallowRender              | component        |  Shallow renders the component using react-test-renderer and returns the render output |
 | shouldShallowMatchSnapshot | title, component | Shallow renders the component using react-test-renderer and asserts that it matches the given snapshot, which will be saved in the `__snapshots__` directory. The first argument `title` is the title for the test. |
 | isNull                     | title, component    | Renders the component using react-test-renderer, converts it to JSON and asserts that it is null. The first argument `title` is the title for the test. |
+| testUtilityPackages                     | actualExports, expectedExports, utilityName | Compares an imported utility package and compares the exported values against a keyed object which validates that all the utilities exports are explicit. |
```

Comment:
```suggestion
| testUtilityPackages        | actualExports, expectedExports, utilityName | Validates an imported utility package's exported values against an object of key-value pairs in the form `{ name_of_export: 'type of export' }`, e.g. `{ shouldMatchSnapshot: 'function' }`. |
```

Reply:
(really just wanted to fix up the spacing to line up with the `|` above it - but had a tinker with the wording while I was there! 😇 )